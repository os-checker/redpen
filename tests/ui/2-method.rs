pub fn panic() {
    panic!("ðŸ’¥")
}

struct S {}
impl S {
    pub fn caller(&self) {
        panic();
        panic!("Second panic.")
    }
}
