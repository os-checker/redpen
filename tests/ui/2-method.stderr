[1m[91merror[0m[1m: A possible panic spot is found.[0m
 [1m[94m--> [0m$DIR/2-method.rs:2:12
  [1m[94m|[0m
[1m[94m1[0m [1m[94m|[0m pub fn panic() {
  [1m[94m|[0m [1m[94m--------------[0m [1m[94mFor this function.[0m
[1m[94m2[0m [1m[94m|[0m     panic!("ðŸ’¥")
  [1m[94m|[0m            [1m[91m^^^^[0m [1m[91mThis may panic![0m
[1m[91merror[0m[1m: A possible panic spot is found.[0m
 [1m[94m--> [0m$DIR/2-method.rs:8:9
  [1m[94m|[0m
[1m[94m7[0m [1m[94m|[0m pub fn caller(&self) {
  [1m[94m|[0m [1m[94m--------------------[0m [1m[94mFor this function.[0m
[1m[94m8[0m [1m[94m|[0m         panic();
  [1m[94m|[0m         [1m[91m^^^^^^^[0m [1m[91mThis may panic![0m
[1m[94m9[0m [1m[94m|[0m         panic!("Second panic.")
  [1m[94m|[0m                [1m[91m^^^^^^^^^^^^^^^[0m [1m[91mThis may panic![0m
